tests/nightly/test_large_array.py

```diff
@@ -279,6 +279,19 @@ def test_diag():
     assert_almost_equal(r.asnumpy(), np.diag(a_np, k=k))
 
 
+def test_transpose():
+    a = nd.arange(0, LARGE_X).reshape(LARGE_X, 1)
+    b = nd.broadcast_to(a, shape=(a.shape[0], SMALL_Y))
```

Comment:
Why not just create a 2D array instead?

Reply:
easier to test this way much convenient to understand when you don't want the test to take monger time to verify each and every element